lib/Dependency.js

```diff
@@ -7,7 +7,7 @@ function Dependency() {
 }
 module.exports = Dependency;
 
-Dependency.prototype.isEqualResource = function( /* other */ ) {
```

Comment:
I wonder why `no-unused-vars` is disabled...

But the argsIgnorePattern seem to be a good option here.